bundles/org.openhab.binding.qbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,266 @@
+		</channels>
+		<config-description>
+			<parameter name="rolId" type="integer" required="true">
+				<label>Rol ID</label>
+				<description>Qbus rol Id</description>
+				<advanced>false</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="thermostat">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Thermostat</label>
+		<description>Qbus thermostat</description>
+		<channels>
+			<channel id="measured" typeId="measured"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="setpoint" typeId="setpoint"/>
+		</channels>
+		<config-description>
+			<parameter name="thermostatId" type="integer" required="true">
+				<label>Thermostat ID</label>
+				<description>Qbus IP Interface Thermostat ID</description>
+				<advanced>false</advanced>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="scene">
+		<item-type>Switch</item-type>
+		<label>Scene</label>
+		<description>Scene control for Qbus</description>
+		<category>Scene</category>
+	</channel-type>
+
+	<channel-type id="co2">
+		<item-type>Number</item-type>
+		<label>CO2</label>
+		<description>CO2 value for Qbus</description>
+		<category>CO2</category>
+	</channel-type>
+
+	<channel-type id="switch">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Switch control for output in Qbus</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<channel-type id="brightness">
+		<item-type>Dimmer</item-type>
+		<label>Brightness</label>
+		<description>Brightness control for dimmer function in Qbus</description>
+		<category>DimmableLight</category>
+	</channel-type>
+
+	<channel-type id="color">
+		<item-type>Color</item-type>
+		<label>Color</label>
+		<description>DMX control for dimmer function in Qbus</description>
+		<category>HSB</category>
+	</channel-type>
+
+	<channel-type id="measured">
+		<item-type>Number:Temperature</item-type>
+		<label>Measured</label>
+		<description>Temperature measured by thermostat</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>CurrentTemperature</tag>
+		</tags>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="setpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Setpoint</label>
+		<description>Setpoint temperature of thermostat</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>TargetTemperature</tag>
+		</tags>
+		<state min="0" max="100" step="0.5" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="mode">
+		<item-type>Number</item-type>
+		<label>Mode</label>
+		<description>Thermostat mode</description>
+		<category>Number</category>
+		<state>
+			<options>
+				<option value="0">Manueel</option>
+				<option value="1">Vorst</option>
+				<option value="2">Economisch</option>
+				<option value="3">Comfort</option>
+				<option value="4">Nacht</option>
```

Comment:
These should be in English. You could add a translation in the file I asked you to remove or fill.